### PR TITLE
Telemetry

### PR DESCRIPTION
**Pull request by DanteSGarrett**
## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Associated Issue(s)
#29 

## Steps to Test
See the readme in telemetry branch

## Changelog
Added telemetry for button inputs of 2 controllers, odometry, and drive train variables.

## What?
Added telemetry for button inputs of 2 controllers, odometry, and drive train variables.

## Why?
People wanted telemetry, so I did that.

### CODE REVIEW

**Review by ChrisHirsch — DISMISSED** [review]
+1Thumb

Good enough for now

**Review by ChrisHirsch — DISMISSED** [review]
+1 Can't wait to see this working!

**Review by raylmorris — DISMISSED** [review]
@DanteSGarrett please create a task for moving this from periodic framework to commands framework

File: swerve/swerveModule.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 import math
```
Comment (tuffinmuffin) [inline swerve/swerveModule.py:1]: Was this file cleaned up on purpose? I see no functional changes?

**Review by DarthCooper — APPROVED** [review]
+1

File: data/telemetry.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
+import wpilib
```
Comment (tuffinmuffin) [inline data/telemetry.py:1]: May want to seperate the drive train telem from other telem and name each in accordance to what they do or what robot they support. Could do this in a refactor later

**Review by tuffinmuffin — DISMISSED** [review]
Left a couple comments should be reviewed and resolved or addressed no show stoppers for now

File: swerve/swerveModule.py (hunk 1 above)
Comment (DanteSGarrett) [reply swerve/swerveModule.py:1]: was not intentional, syncing code with main and other branches may have caused that to happen.

File: data/telemetry.py (hunk 2 above)
Comment (jtdurham) [reply data/telemetry.py:1]: Noted, will add to Kanban board

**Review by raylmorris — APPROVED** [review]
+1

**Review by jtdurham — APPROVED** [review]
+1, nice work!!

**Pull request merged**